lib/auth/Kconfig

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+
+menuconfig AUTH_LIB
+	bool "Authentication library"
+	help
+	  This option enables the Authentication library.
+
+if AUTH_LIB
+	config AUTH_CHALLENGE_RESPONSE
+		bool "Challenge Response (Must use TINYCRYPT and TINYCRYPT_SHA256)"
+		help
+		Selects the Challenge Response authentication method.
+		Must use TINYCRYPT and TINYCRYPT_SHA256
+
+	config AUTH_DTLS
+		bool "DTLS Authentication"
+		help
+		Selects the DTLS authentication method.
+
```

Comment:
something here, this code depends on mbedtls doesn't ?

Reply:
Fixed.